cmake/modules/packages/FindPoppler.cmake

```diff
@@ -134,6 +134,14 @@ find_package_handle_standard_args(Poppler
                                   HANDLE_COMPONENTS)
 mark_as_advanced(Poppler_INCLUDE_DIR Poppler_LIBRARY)
 
+if(Poppler_FOUND)
+  if (NOT EXISTS "${Poppler_INCLUDE_DIR}/Object.h")
+    message(WARNING "Poppler private headers not found. Make sure you build Poppler with -DENABLE_UNSTABLE_API_ABI_HEADERS")
+    unset(Poppler_FOUND)
+    unset(POPPLER_FOUND)
```

Comment:
Just curious, why is it necessary to check for upper cased?

Reply:
cf documentation of FOUND_VAR in https://cmake.org/cmake/help/latest/module/FindPackageHandleStandardArgs.html. It as needed because CMake GDAL code later seemed to test POPPLER_FOUND